src/schema/schema.graphql

```diff
@@ -273,6 +273,7 @@ enum ZimletPresence {
 
 type AuthResponse {
 	authToken: [AuthToken]
+	twoFactorAuthRequired: TwoFactorAuthRequired
```

Comment:
as per docs, type of this property should be Boolean, it's weird that current setup works

Reply:
anyways api response confirms to implementation, no need to anything here

Reply:
Updated.